newspaper/nlp.py

```diff
@@ -9,17 +9,34 @@
 
 import re
 import math
+from os import path
 
 from collections import Counter
 
 from . import settings
 
-with open(settings.NLP_STOPWORDS_EN, 'r') as f:
-    stopwords = set([w.strip() for w in f.readlines()])
-
 ideal = 20.0
 
+stopwords = set()
 
+def load_stopwords(language):
```

Comment:
One interesting side-effect of you moving the stopwords loading code from the global scope into a function is that the stopwords loading is now going to be run dynamically when `Article.nlp()` is called .. instead of being run at import time.

I am not sure if this side-effect will consequences

Reply:
Actually, after merging your change I'm going to commit a change which moves all of the `load_stopwords` code into **global scope** and also deletes the `nlp.load_stopwords(self.config.get_language())` in article.py to minimize side effects.